src/managers/run-strategies/persistent-single-player-strategy.js

```diff
@@ -0,0 +1,85 @@
+'use strict';
+
+var classFrom = require('../../util/inherit');
+var IdentityStrategy = require('./identity-strategy');
+var StorageFactory = require('../../store/store-factory');
+var StateApi = require('../../service/state-api-adapter');
+
+var keyNames = require('../key-names');
+
+var defaults = {
+    store: {
+        synchronous: true
+    }
+};
+
+var Strategy = classFrom(IdentityStrategy, {
+    constructor: function Strategy(runService, options) {
+        this.run = runService;
+        this.options = $.extend(true, {}, defaults, options);
+        this.runOptions = this.options.run;
+        this._store = new StorageFactory(this.options.store);
+        this.stateApi = new StateApi();
+
+        this._loadAndCheck = this._loadAndCheck.bind(this);
+        this._restoreRun = this._restoreRun.bind(this);
+        this._getAllRuns = this._getAllRuns.bind(this);
+        this._loadRun = this._loadRun.bind(this);
+    },
+
+    reset: function (runServiceOptions) {
+        return this.run
+            .create(this.runOptions, runServiceOptions)
+            .then(function (run) {
+                run.freshlyCreated = true;
+                return run;
+            });
+    },
+
+    getRun: function () {
+        return this._getAllRuns()
+            .then(this._loadAndCheck);
+    },
+
+    _getAllRuns: function () {
+        var session = JSON.parse(this._store.get(keyNames.EPI_SESSION_KEY) || '{}');
+        return this.run.query({
+            'user.id': session.userId || '0000',
```

Comment:
I wanted the query string to look 'incorrect' as opposed to just "userId=" which filters out all users, but is doesn't look evil at first glance :)